prov/rxm/src/rxm.h

```diff
@@ -127,18 +131,22 @@ struct rxm_rma_iov {
 	struct ofi_rma_iov iov[];
 };
 
-/* States for large message transfer */
-#define RXM_LMT_STATES(FUNC)	\
-	FUNC(RXM_LMT_NONE),	\
-	FUNC(RXM_LMT_START),	\
-	FUNC(RXM_LMT_ACK),	\
+/* RXM protocol states / tx/rx context */
+#define RXM_PROTO_STATES(FUNC)	\
+	FUNC(RXM_NONE),		\
+	FUNC(RXM_TX),		\
+	FUNC(RXM_RX),		\
+	FUNC(RXM_LMT_TX),	\
+	FUNC(RXM_LMT_ACK_WAIT),	\
+	FUNC(RXM_LMT_READ),	\
+	FUNC(RXM_LMT_ACK_SENT), \
 	FUNC(RXM_LMT_FINISH),
 
-enum rxm_lmt_state {
-	RXM_LMT_STATES(ENUM)
+enum rxm_proto_state {
+	RXM_PROTO_STATES(ENUM)
```

Comment:
I realize this is just a rename, but I don't understand this sort of syntax for creating an enum.